### PR TITLE
Convert to rizin APIs

### PR DESCRIPTION
**Pull request by b1llow**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [x] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [ ] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

Convert:

- `es`
- `il`
- `drp`
- `ahts`
- `.ar-`
- `wcr`
- `aets-`
- `e <val>=?`
- `fo`

And add

- `CutterCore.getConfigVariableSpaces`
- `CutterCore.getConfigOptions`

**Test plan (required)**

**Closing issues**

Partially addresses https://github.com/rizinorg/cutter/issues/2666

### CODE REVIEW

File: src/core/Cutter.cpp [hunk 1]
```diff
@@ -4430,3 +4457,23 @@ QByteArray CutterCore::ioRead(RVA addr, int len)
 
     return array;
 }
+
+QStringList CutterCore::getConfigVariableSpaces(const QString &key)
```
Comment (XVilka) [inline src/core/Cutter.cpp:4461]: Eventually, we might want to provide this from the Rizin side of the API.

**Review by XVilka — APPROVED** [review]

File: src/core/Cutter.cpp [hunk 2]
```diff
@@ -2146,11 +2160,15 @@ void CutterCore::stopDebug()
     currentlyRemoteDebugging = false;
     emit debugTaskStateChanged();
 
+    CORE_LOCK();
     if (currentlyEmulating) {
-        cmdEsil("aeim-; aei-; wcr; .ar-; aets-");
+        cmdEsil("aeim-; aei-;");
```
Comment (wargio) [inline src/core/Cutter.cpp]:
```suggestion
        cmdEsil("aeim- ; aei-");
```

File: src/core/Cutter.cpp (hunk 1 above)
Comment (b1llow) [reply src/core/Cutter.cpp:4461]: It might be a separate PR.

Comment (XVilka) [reply src/core/Cutter.cpp:4461]: Yes, that's why I wrote "eventually".

File: src/core/Cutter.cpp [hunk 3]
```diff
@@ -2146,11 +2160,15 @@ void CutterCore::stopDebug()
     currentlyRemoteDebugging = false;
     emit debugTaskStateChanged();
 
+    CORE_LOCK();
     if (currentlyEmulating) {
-        cmdEsil("aeim-; aei-; wcr; .ar-; aets-");
```
Comment (ret2libc) [inline src/core/Cutter.cpp:2150]: I can't find `ar-` anymore... Where did you get it?

**Review by wargio — APPROVED** [review]

File: src/core/Cutter.cpp (hunk 3 above)
Comment (XVilka) [reply src/core/Cutter.cpp:2150]: Looks like it was removed. But it should be the right matching function:
- https://github.com/rizinorg/rizin/blob/dev/librz/core/cmd_descs/cmd_debug.yaml#L712
- https://github.com/rizinorg/rizin/commit/d04e2a805c982bbdf817fd42c4905cf6bf17397c

**Pull request merged**